webpack.common.js

```diff
@@ -1,26 +1,51 @@
-const path = require("path");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const Dotenv = require("dotenv-webpack");
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const Dotenv = require('dotenv-webpack');
 
 module.exports = {
-  entry: path.join(__dirname, "src", "index.js"),
+  entry: path.join(__dirname, 'src', 'index.js'),
+  resolve: {
+    modules: [path.resolve(__dirname, 'node_modules'), 'node_modules'],
+    extensions: ['.js', '.jsx'],
+    symlinks: false,
+    alias: {
+      '@': path.resolve(__dirname, 'src'),
+      Images: path.resolve(__dirname, 'src', 'Images'),
+      Components: path.resolve(__dirname, 'src', 'Components'),
```

Comment:
folder name components is lower case

Reply:
in the files the Components folder uses a capital C in the start. Should it have a lowercase c instead?

Reply:
Answered above